src/constants/collection/index.ts

```diff
@@ -16,6 +16,7 @@ export enum CollectionSummaryType {
     trash = 'trash',
     all = 'all',
     shared = 'shared',
+    sharedViaLink = 'sharedViaLink',
```

Comment:
`sharedOnlyViaLink` name would be more clear (for what we are doing) and avoid any  confusion 

refer comment on `isSharedViaLink` for the reason